app/poros/item.rb

```diff
@@ -0,0 +1,14 @@
+class Item
+  attr_reader :id,
+              :name,
+              :description,
+              :unit_price,
+              :merchant_id
+  def initialize(data)
+    @id           = data[:id]
+    @name         = data[:attributes][:name]
+    @description = data[:attributes][:description]
+    @unit_price   = data[:attributes][:unit_price]
+    @merchant_id  = data[:attributes][:merchant_id]
+  end
+end
```

Comment:
This comment goes for this class and the `merchant.rb` class - careful naming attributes "id", since POROs automatically get a unique id assigned to them. This could cause some conflicts later on.